src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
C# as of now is fine with sync client, async client, async server and async generic server.